Fabric.Authorization.API/Modules/RootModule.cs

```diff
@@ -11,7 +12,7 @@ public RootModule(): base("/v1")
 
         private dynamic Redirect()
         {
-            return Response.AsRedirect("/client/index.html");
+            return Response.AsRedirect($"/{AccessControl.Path}/index.html");
```

Comment:
Will this allow us to hit the app using https://host.domain.local/Authorization, or do we have to enter:
https://host.domain.local/Authorization/accesscontrol?

Reply:
Yes, when I run locally and go to `localhost:5004`, I get redirected to the landing page for the access control UI.